hoomd/mpcd/test-py/make_random_sphere_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2009-2019 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+# Maintainer: mphoward
+
+import unittest
+import hoomd
+import numpy as np
+from hoomd import mpcd
+
+# unit tests for generating random solvent particles inside sphere
+class mpcd_snapshot(unittest.TestCase):
+    def setUp(self):
+        hoomd.context.initialize()
+        hoomd.init.read_snapshot(hoomd.data.make_snapshot(N=0, box=hoomd.data.boxdim(L=120.)))
+
+    def test_set_density(self):
```

Comment:
This test name isn't accurate
```suggestion
    def test_init(self):
```